author/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_reader :first_name, :last_name
+
+  def initialize(first_name, last_name, id = Random.rand(1..1000))
+    @first_name = first_name
+    @last_name = last_name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
```

Comment:
Great work on this project milestone.

I think that line 13 should come after line 12 because the item pushed into the @items array will be an item without author defined since it was pushed in before author is set.

Reply:
Great, Thanks